app/models/project.rb

```diff
@@ -210,12 +210,16 @@ def with_push
       joins(:events).where('events.action = ?', Event::PUSHED)
     end
 
+    def without_archived
+      where('projects.archived = ?',false)
+    end
+
     def active
       joins(:issues, :notes, :merge_requests).order('issues.created_at, notes.created_at, merge_requests.created_at DESC')
     end
 
     def search(query)
-      joins(:namespace).where('projects.archived = ?', false).
+      joins(:namespace).
```

Comment:
I found it in the `app/controllers/explore/projects_controller.rb` and added the non_archived scope